massa-bootstrap/src/bindings.rs

```diff
@@ -0,0 +1,56 @@
+mod client;
+mod server;
+use std::{
+    io::{self, ErrorKind},
+    time::{Duration, Instant},
+};
+
+pub(crate) use client::*;
+pub(crate) use server::*;
+
+trait BindingReadExact: io::Read {
```

Comment:
for a side-by-side view of this diff, you can exclude the fmt and module re-arrangement commit.